recipe/meta.yaml

```
@@ -41,6 +42,7 @@ requirements:
     - versioneer
     - meson =1.2.1
     - meson-python =0.13.1
```

I think meson needs to be unpinned here
```suggestion
    - meson
    - meson-python
```